Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/FeedFragment.java

```diff
@@ -49,9 +67,37 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         feedAdapter = new FeedListAdapter(getContext(), feedDataList);
         feedListView.setAdapter(feedAdapter);
 
+        userService = new UserService();
+
         return rootView;
     }
 
+    /**
+     * Initialize the contents of the Activity's standard options menu.
+     * @param menu The options menu in which you place your items.
+     * @param inflater The MenuInflater object that can be used to inflate any itmes in the menu
+     */
+    @Override
+    public void onCreateOptionsMenu(@NonNull Menu menu, @NonNull MenuInflater inflater) {
```

Comment:
Is it possible at all to have this stuff in the main activity or no? A lot of the same code is repeated.

Reply:
Most of the duplicate code has been moved into the main activity